core/server/api/v2/utils/serializers/input/pages.js

```diff
@@ -8,6 +8,18 @@ function removeMobiledocFormat(frame) {
     }
 }
 
+function setDefaultOrder(frame) {
+    let includesOrderedRelations = false;
+
+    if (frame.options.withRelated) {
```

Comment:
We have dafault ordering by when [authors](https://github.com/TryGhost/Ghost/blob/master/core/server/models/post.js#L480) or [tags](https://github.com/TryGhost/Ghost/blob/master/core/server/models/post.js#L485) are present in the include. Didn't find this behavior being described explicitly anywhere in the API Docs. Should we add it?

Reply:
Ah this is the sort order of the things themselves related to the posts right?

There is also a change in order of posts when you do a filter with an in query.. or there should be?

Reply:
> Ah this is the sort order of the things themselves related to the posts right?

Yes, exactly.

>There is also a change in order of posts when you do a filter with an in query.. or there should be?

This case is handled with `!frame.options.order` below, if I understand the question correctly

Reply:
I linked to wrong ordering source above these are correct ones for [tags](https://github.com/TryGhost/Ghost/blob/master/core/server/models/post.js#L648) & [authors](https://github.com/TryGhost/Ghost/blob/master/core/server/models/post.js#L653). Which also leads to correct the check, needs to account for  `tag` and `author`

Reply:
The `IN` filter special order is supposed to order posts by how many of the requested tags they match. 

So if I ask for posts with one of 3 tags, a post that has all 3 would appear ahead of a post that has only 1. 

What's actually happening, I think, is that they're being ordered by how many tags they have in total 🤔 

Nothing to fix rn but something to maybe raise?

Reply:
Raised an issue here https://github.com/TryGhost/Ghost/issues/10377